index.js

```diff
@@ -0,0 +1,166 @@
+//Napisz program, który będzie symulować działanie pralki (takiej do prania ubrań)
+//Twoja pralka powinna umożliwiać użytkownikowi wybranie programu prania (szybkie, długie)
+//Powinna umieć wirować (na koniec szybkiego prania, oraz na koniec i w połowie długiego)
+//Po wybraniu jakiegokolwiek programu powinna odpompować starą wodę, pobrać nową.
+//Na podstawie wybranego programu podgrzać każdorazowo wodę do odpowiedniej temperatury.
+//I odpompować ją przed wirowaniem.Powinna umożliwiać użytkownikowi uruchomić wirowanie na żądanie.
+//Aby zabezpieczyć łazienkę przed zalaniem, pralka powinna ryglować drzwi pralki po włączeniu
+//jakiegokolwiek programu oraz odblokowywać 30 sekund po skończonym praniu.
+//Pralka powinna płukać ubrania przed wirowaniem, oraz udostępniać płukanie z wirowaniem jako
+//osobny program do wybrania.
+//Każda pralka bierze proszek, więc programy piorące powinny sprawdzać czy użytkownik
+//wsypał proszek do prania.
+//Pralka powinna sprawdzać czy użytkownik włożył ciuchy do bębna.
+// W przypadku ich braku powinna poinformować użytkownika o tym.
+
+class WashingmMachine {
+  constructor() {
+    this._program = null;
```

Comment:
Każdy program to osobny byt, pralka tylko używa programów do swojej pracy. Każdy program powinien być osobną klasą, która zawiera mechanizm prania/wirowania